lib/rubocop/cop/style/file_null.rb

```diff
@@ -56,6 +63,14 @@ def on_str(node)
 
         private
 
+        def contain_dev_null_string_in_file?
```

Comment:
This can be done once in `on_new_investigation`, right? Seems needlessly expensive to do for every string.